### PR TITLE
[Tests-Only]Refactor the capabilities test file to use provider verify

### PR DESCRIPTION
**Pull request by jasson99**
## Description:
This PR refactors the capabilities test file such that only the required interactions are imported. Also, verification of the requests and the interactions is done.

## Related Issue:
- Part of #643

### CODE REVIEW

**Comment by jasson99** [comment]
The interactions for which functions are created and exported are not removed yet, because they are still used in other test files. Once we refactor all or almost all the test files, we can get rid of the redundancy.

**Comment by HariBhandari07** [comment]
LGTM

File: tests/capabilitiesTest.js [hunk 1]
```diff
@@ -8,12 +8,52 @@ describe('Main: Currently testing getConfig, getVersion and getCapabilities', fu
...
+      auth: {
+        basic: {
+          username: config.username,
+          password: config.password
+        }
+      }
+    })
+    await oc.login()
+    return oc
+  }
```
Comment (individual-it) [inline tests/capabilitiesTest.js:32]: this is repeated in every file, can we move it to the helper?

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/capabilitiesTest.js (hunk 1 above)
Comment (jasson99) [reply tests/capabilitiesTest.js]: fixed

**Review by individual-it — APPROVED** [review]

**Pull request merged**